code/modules/mob/living/damage_procs.dm

```diff
@@ -207,15 +207,15 @@
 	return cloneloss
 
 /mob/living/proc/adjustCloneLoss(amount, updating_health = TRUE, forced = FALSE)
-	if(!forced && (status_flags & GODMODE) || HAS_TRAIT(src, TRAIT_NOCLONELOSS))
+	if(!forced && ((status_flags & GODMODE) || HAS_TRAIT(src, TRAIT_NOCLONELOSS)))
 		return FALSE
 	cloneloss = CLAMP((cloneloss + (amount * CONFIG_GET(number/damage_multiplier))), 0, maxHealth * 2)
 	if(updating_health)
 		updatehealth()
 	return amount
 
 /mob/living/proc/setCloneLoss(amount, updating_health = TRUE, forced = FALSE)
-	if(!forced && (status_flags & GODMODE) || HAS_TRAIT(src, TRAIT_NOCLONELOSS))
+	if(!forced && ((status_flags & GODMODE) || HAS_TRAIT(src, TRAIT_NOCLONELOSS)))
```

Comment:
triple parenthesis?

Reply:
Yeah. We're checking that it isn't forced AND that the mob isn't in god mode OR has the trait. We want to check that it isn't forced in both cases.

Reply:
ah, didnt see the two parenthesis inbetween